maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -310,6 +353,44 @@ private void downloadOWAs(File targetDirectory, DistroProperties distroPropertie
 		}
 	}
 
+	private void setConfigFolder(Server server, DistroProperties distroProperties) throws MojoExecutionException {
+		if (distroProperties != null) {
+			File configDir = new File(server.getServerDirectory(), SDKConstants.OPENMRS_SERVER_CONFIGURATION);
+			configDir.mkdir();
+			downloadConfigs(distroProperties, configDir);
+			File referenceApplicationFile = new File(configDir, "referenceapplication-distro.owa");
+			if(!referenceApplicationFile.exists()) {
+				return;
+			}
+			try {
+				ZipFile zipFile = new ZipFile(referenceApplicationFile);
+				zipFile.extractAll(configDir.getPath());;
```

Comment:
Is the double semi colon a new syntax? 😊

Reply:
😅